src/zod/update-password.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { z, type ZodType } from 'zod';
+import { type ProfileFormData } from '#/modules/users/types';
+
+export const UpdatePasswordSchema: ZodType<Partial<ProfileFormData>> = z
+  .object({
+    oldPassword: z
+      .string()
+      .min(8, { message: 'Entered password is too short' })
+      .max(255, { message: 'Entered password is too long' }),
```

Comment:
Do not validate the length of the old password.

Reply:
fixed